projects/docs-tools/src/modules/code-examples/code-examples-editor.service.ts

```diff
@@ -144,22 +161,23 @@ export class SkyDocsCodeExamplesEditorService {
  `;
 
     const moduleImportStatements: string[] = [
-      `import { NgModule } from '@angular/core';`,
       `import { FormsModule, ReactiveFormsModule } from '@angular/forms';`,
-      `import { BrowserAnimationsModule } from '@angular/platform-browser/animations';`,
-      `import { RouterModule } from '@angular/router';`,
-      `import { provideInitialTheme } from '@skyux/theme';`,
-      `import { AppComponent } from './app.component';`,
     ];
 
-    const moduleImports: string[] = [
-      'BrowserAnimationsModule',
-      'FormsModule',
-      'ReactiveFormsModule',
-      'RouterModule.forRoot([])',
-    ];
+    const moduleImports: string[] = ['FormsModule', 'ReactiveFormsModule'];
 
-    const files: { [_: string]: string } = {};
+    const files: { [_: string]: string } = {
+      '.stackblitzrc': JSON.stringify(
+        {
+          env: {
+            NG_BUILD_PARALLEL_TS: '0',
+            NODE_ENV: 'development',
+          },
+        },
+        undefined,
+        2,
+      ),
+    };
```

Comment:
[Angular.dev](https://angular.dev/) uses these settings for their Web Containers.